internal/server/data/data.go

```diff
@@ -92,6 +92,16 @@ func (d *DB) QueryRow(query string, args ...any) *sql.Row {
 	return d.DB.Raw(query, args...).Row()
 }
 
+func (d *DB) OrganizationID() uid.ID {
+	// FIXME: this is a hack to keep our tests passing. The db should not
+	// be scoped to an org ID.
+	return d.DefaultOrg.ID
```

Comment:
Calling this out. This is not a regression, because it works this way on `main`, but it is a problem we should fix.

All of our test fixtures should be more explicit about which org they are using when creating things.

Reply:
Exactly. This is why I prioritized this part of the `data` package work, because this is the behaviour we have now by setting the context when we create a database.

It "works" in production because requests use the transaction reference where we override the organization value. 

The only code that doesn't do that right now is:
* the config loading, where you fixed it to set this same default value
* tests in the `server` package where we do the same thing by setting the default org.

I plan on removing this in the first follow up PR.